lib/wahwah/tag.rb

```diff
@@ -27,25 +27,30 @@ class Tag
     )
 
     def initialize(file)
```

Comment:
Since we already have handle params in WhaWah.open, this initialize method should only accept IO as params.

Reply:
I allowed it to accept both because I didn't want to encroach on too much code, and the tests often use, for example, `Mp3Tag.new "example.mp3"`. Would you like me to implement this and rewrite the tests to use `WahWah::open` (or `XxxTag::new` with an IO, when testing specific `Tag` subclasses)?

Reply:
I used to allow `Tag` class to accept IO like params mainly because in some tests I need to pass IO object to make test easy. So now we don't need to handle file path in `Tag` initialize method anymore, I think just accepting the IO is better. And for testing specific tag subclasses, you can `File.open` file path can pass IO object to the class. I know there are a lot of places in the test need to change. It's very tedious. I hope you don't mind.